gloo-edge/gloo-edge/README.md

```diff
@@ -567,26 +1028,53 @@ spec:
 EOF
 ```
 
-The rule means that a username can only contain letters.
+Refresh the **httpbin** application and after 5 times, you will retrieve a **429 error**. This is because you are an anonymous user.
 
-Click on the `Sign in` button and try to login with a user named `user1` (the password doesn't matter).
+```
+/opt/google/chrome/chrome $(glooctl proxy url --port https)/not-secured/get
```

Comment:
I think you should continue to use `curl` here

Reply:
something like this?

```
for i in {1..8}
do
curl -k -s -o /dev/null -w "%{http_code}" $(glooctl proxy url --port https)/not-secured/get
done
```